chapters/001-getting-started/README.md

```diff
@@ -123,22 +142,20 @@ command: `docker network inspect $NETWORK_NAME | jq ".[].Containers"`
 
 ### Interlude: What is going on here so far?
 
-Okay, so we have two containers that we've started up and put onto a new docker-defined network. You'll notice that these containers have IP addresses already (the above `network inspect` command shows them; also, on the bash session with each container, if you type `ip addr`, it will show you the addresses on the machines).
+Okay, so we have two containers that we've started up and put onto a new docker-defined network. You'll notice that these containers have IP addresses already. The above `network inspect` command shows them; also, on the bash session with each container, if you run the `ip addr` command, it will show you the addresses on the machines (shown in a few paragraphs).
```

Comment:
side note: we *may* want to present commands in un-abbreviated format. In other words, the **actual** command here is `ip address`, not `ip addr`. 

There is a bit of a tradition within networking CLIs to allow users to abbreviate commands (cisco CLIs are famous for this), and the `ip` command carries this forward. But, it may be best to leave it unabbreviated for the uninitialized human.